app/models/ability.rb

```diff
@@ -1,8 +1,5 @@
-# frozen_string_literal: true
-
 class Ability
   include CanCan::Ability
-
   def initialize(user)
     # Define abilities for the user here. For example:
     #
```

Comment:
- [ ] `app/views/devise/sessions/new.html.erb` file: As required for the project, kindly make sure you have modified the views in devise for the **login page**to match the **[wireframes and their styling](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)** and also remember to fix the broken image at your login page.
![Screenshot from 2023-04-21 23-19-44](https://user-images.githubusercontent.com/83042742/233737577-b3c26434-b805-4460-a3ab-8793d4645ae8.png)